### ISSUE TITLE
diff / patch syntax highlighting

### ISSUE THREAD

**Issue by dsernst** [npm/marky-markdown#64]
Reposted from https://github.com/npm/newww/issues/1432

----------------

GitHub supports `diff` syntax highlighting:

```diff
old line
old line
+ insert line
old line
- remove line
old line
```

But [npmjs.com](http://npmjs.com) does not. :cry:

**Comment by ashleygwilliams** [comment]
fixed in #84

**Comment by dsernst** [comment]
:sunglasses:
